### DISCUSSION TITLE
방탈출 리서치

### DISCUSSION THREAD

**Discussion by wooyn730** [GG-Studio-990001/Grayed-Game#80] [Ideas]
Discussion에 이런거 올려도 되는지 모르겠지만..
함께 재밌게 방탈출 할 날을 기다리며..😇🤟

[방탈출 리서치 파일](https://docs.google.com/spreadsheets/d/1doNRz1rM8A0qBkQG4m7UAfJLneiYe1nYIwa_XeyjaGw/edit#gid=0)

**Comment by fkdl0048** [comment]
오오... 저는 개인적으로 스토리 있는 방탈출 좋아합니다 ㅎ..

**Reply by wooyn730** [reply]
저도 스토리도 진짜 좋아하는데 6명이서 할 만한 방탈출 자체가 넘 적네요..🥹

개유명한 방탈출도 같이 해보고 싶슫니당
